### PR TITLE
Add command to access Vault console

### LINKED ISSUE #184: Add "h1 vault stop" & "h1 vault start"
And update guide

### PR DESCRIPTION
**Pull request by ad-m**
Fix #184

### CODE REVIEW

File: bin/vault/console/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const genericConsole = require('bin/generic/console');
+
+module.exports = resource => genericConsole(resource,
+    (args, vault) => `https://panel.hyperone.com/console/vault/${vault._id}?tenant=${args.profile.project._id}&scope=${process.env.SCOPE_NAME}`
```
Comment (fredericosilva) [inline bin/vault/console/index.js]: Use project query parameter instead of tenant.

Comment (ad-m) [reply bin/vault/console/index.js]: This will corrupt the functionality of the application if you do not make changes on your side.

Comment (ad-m) [reply bin/vault/console/index.js]: Fixed.

Comment (fredericosilva) [reply bin/vault/console/index.js]: fixed on my side

Comment (ad-m) [reply bin/vault/console/index.js]: Cool! Ready to merge!

**Comment by ad-m** [comment]
:fireworks:

**Pull request merged**